src/Bicep.Cli.IntegrationTests/BuildParamsCommandTests.cs

```diff
@@ -110,8 +110,40 @@ public async Task Build_Params_With_CLI_Input_And_Using_Declaration_Bicep_File_R
             result.Should().Be(1);
             output.Should().BeEmpty();
             error.Should().Contain($"Bicep file {otherBicepPath} provided with --bicep-file option doesn't match the Bicep file {bicepPath} referenced by the using declaration in the parameters file");
+            // error
         }
 
+
```

Comment:
[nit]: There are some extra newlines (line 116, line 138, and lines 145-146) that should be deleted.